addons/point_of_sale/static/src/app/error_handlers/error_handlers.js

```diff
@@ -55,6 +55,10 @@ registry.category("error_handlers").add("offlineErrorHandler", offlineErrorHandl
 
 function defaultErrorHandler(env, error, originalError) {
     originalError = identifyError(originalError);
+    // Assume that the unhandled falsey rejections can be ignored.
+    if (!originalError) {
+        return true;
+    }
```

Comment:
Empty rejections that are ignored are generally a bad pattern. From the PoV of browsers, a rejection *is* an error, it shouldn't be used as control flow. It's like throwing undefined. If you want the code to stop proceding, you can return a boolean, check that boolean and bail if falsy. If for some reason this isn't feasible, you should still prefer returning a pending promise instead, which will effectively do the same thing (it will prevent whatever is chained after the promise from being executed) but won't invoke all the error handling machinery and will avoid needing a special case in the error handling code for something that's not expected to be treated as an error.

Reply:
Thank you for the explanation.
I changed the implementation to use boolean, tell me what you think :)

Reply:
Seems good :+1: nit: the `Promise.resolve` is useless since this is an async function.